### ISSUE TITLE
Error login query_id 

### ISSUE THREAD

**Issue by SenpaiSeeker** [MeoMunDep/Goats#1]
[10/29/2024, 18:30:33] - {@MeoMunDep x Goats} - [!] Account 1 | Login failed: Request failed: 403 - Forbidden
[10/29/2024, 18:30:33] - {@MeoMunDep x Goats} - [/] Account 1 | Is reconnecting...

**Comment by MeoMunDep** [comment]
it token bro, i update the file early day.

**Comment by SenpaiSeeker** [comment]
yes it is a new token but an error like that appears

**Comment by MeoMunDep** [comment]
i still can run the script
![Screenshot 2024-10-30 134739](https://github.com/user-attachments/assets/24400859-169c-4ea0-8748-feead791602f)
, i think ur ip have been ban bro

**Comment by SenpaiSeeker** [comment]
I use vps bro

**Comment by MeoMunDep** [comment]
yeah, i still can run it mate, so not the game or mine bot problem

**Comment by SenpaiSeeker** [comment]
I'm from Indonesia, how come my IP is in blacklist

**Comment by MeoMunDep** [comment]
i dont know why u keep asking these question bro, the point is game developer, not me
